python_modules/libraries/dagster-aws/dagster_aws/solids.py

```diff
@@ -79,3 +100,71 @@ def download_from_s3(context):
         s3.download_file(Bucket=bucket, Key=key, Filename=target_file, Callback=logger)
 
     return target_file
+
+
+class EmrRunJobFlowSolidDefinition(SolidDefinition):
```

Comment:
Cool. Perhaps factor out the bulk of the cluster creation into its own function to prep for conversation to a resource, although we can just do it then too

Reply:
yeah, let's do it then